### ISSUE TITLE
Typescript definition mistakenly requires optional `brand` attribute

### ISSUE THREAD

**Issue by dylanb** [dequelabs/axe-core#1439]
Expectation: `axe.configure` calls missing optional attributes should not trigger Typescript lint errors

Actual: calling 

```
        axe.configure({
            branding: {
                application: 'appID'
            }
        });

```
Triggers a Typescript error complaining that `brand` is missing

Motivation: causes people to have to know what the default is and not mess it up

<pre><code>
axe-core version: 3.1.2
</code></pre>

**Comment by dylanb** [comment]
Already fixed in 3.2.0
